frontend/src/common/theme.ts

```diff
@@ -159,7 +159,7 @@ const typography = {
       xxl: {
```

Comment:
Could we increase `header.xxl` to `fontSize: 32px` and `lineHeight: 36px`?

I'd like to do this instead of making `header.xl` smaller.